content/administrators/people/staff.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Thanks Harry! Looks like you've successfully updated and removed one file, and this one tagged along for the ride 😄  When you're committing, you can uncheck files and leave then unstaged (in the bottom part of SourceTree rather than the top) and then they wont be committed. In this case, you may need to delete the file, commit the deletion, and then re-create it after pushing your commit.